docs/specification.md

```diff
@@ -93,6 +95,45 @@ The "Sign In" page includes:
 - "Password" field.
 - "Log In" button: Submits the form and redirects the user to the "Projects" page.
 
+Hide/Show password toggle must be located on the right side of a Password input field. Single-click to expose/hide password. (For both SignIn and SignUp)
+
+Red validation message (‘Field cannot be empty’) appears below the inputs when fields are empty. (for both SignIn, SignUp including Name, Email and Password fields).
+
+When attempting to log in with invalid data a Red validation message appears under the Password filed (‘Invalid login credentials. Please try again’)
```

Comment:
This will be a part of common notification messages on failed requests, so I think this clarification is not needed
```suggestion
```